assets/less/base/sizing.less

```diff
@@ -34,6 +34,13 @@ each(@min-heights, #(@k, @v) {
   }
 });
 
+@media screen and (-webkit-min-device-pixel-ratio:0) {
+	.platform-img {
+		width: 100%;
+		height: 100%;
+	}
+}
+
```

Comment:
No need to add the width: 100%, only the height.
Also, please respect the current file indentation.